apps/prairielearn/assets/scripts/lib/editAssessmentQuestionsTable.ts

```diff
@@ -0,0 +1,374 @@
+                          return questionRow({
+                            question: alternative,
+                            zoneIndex,
+                            questionIndex,
+                            alternativeIndex,
+                            showAdvanceScorePercCol,
+                            assessmentType,
+                            // For the alternative groups, pass in points from the alternative group level rather than off the specific question itself
+                            init_points: question.init_points,
+                            points_list: question.points_list,
+                            max_manual_points: question.max_manual_points,
+                            max_auto_points: question.max_auto_points,
+                          });
+                        })}
+                        <tr>
+                          <td></td>
+                          <td></td>
+                          <td colspan="${nTableCols - 2}">
+                            <button
+                              class="btn btn-sm addQuestion"
+                              data-zone-index="${zoneIndex}"
+                              data-question-index="${questionIndex}"
+                              data-alternative-index="${question.alternatives.length}"
+                            >
+                              <i class="fa fa-add" aria-hidden="true"></i> Add Question to Group
+                            </button>
+                          </td>
+                        </tr>
+                      `
+                    : questionRow({
+                        question,
+                        zoneIndex,
+                        questionIndex,
+                        showAdvanceScorePercCol,
+                        assessmentType,
+                        init_points: question.init_points,
+                        points_list: question.points_list,
+                        max_manual_points: question.max_manual_points,
+                        max_auto_points: question.max_auto_points,
+                      })}
+                `;
+              })}
+              <tr>
+                <td></td>
+                <td colspan="${nTableCols - 1}">
+                  <button
+                    class="btn btn-sm add-question"
+                    data-zone-index="${zoneIndex}"
+                    data-question-index="${zone.questions.length}"
+                  >
+                    <i class="fa fa-add" aria-hidden="true"></i> Add Question to Zone
+                  </button>
+                </td>
+              </tr>
+            `;
+          })}
+          <tr>
+            <td colspan="${nTableCols}">
+              <button class="btn btn-sm add-zone" data-zone-index="${zones.length}">
+                <i class="fa fa-add" aria-hidden="true"></i> Add Zone
+              </button>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  `;
+}
+
+function questionRow({
```

Comment:
Component functions should typically be `PascalCase`.

Reply:
Updated to `PascalCase` in [3c6791f](https://github.com/PrairieLearn/PrairieLearn/pull/10255/commits/3c6791f5b7ef00282d313997acf9af8418da3660).